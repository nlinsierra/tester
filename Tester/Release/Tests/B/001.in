5D TC 7H 9H 3H 3D TD JD 7S 4S 8S 5S KS 6D 6S 8H JS 2D KH
