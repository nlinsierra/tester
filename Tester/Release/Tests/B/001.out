33
2C 3C 4C 5C 6C 7C 8C 9C JC QC KC AC 4D 7D 8D 9D QD KD AD 2H 4H 5H 6H TH JH QH AH 2S 3S 9S TS QS AS
